crypto/keyspb/keyspb.proto

```diff
@@ -0,0 +1,63 @@
+// Copyright 2017 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package keyspb;
+
+// Generator defines how a new pair of asymmetric keys should be generated.
```

Comment:
I think it a bit strange that we keep the Godoc convention in protos. It makes for unusual comments when compared to proto libraries as used by other languages. (Very minor, feel free to push back.)